docs/iot/iot-partners/thingspark.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with ThingsPark Cloud
+
+The ThingsPark service is a cloud-based IoT solution that can be connected with various things. 
+
+Tizen developers can easily develop an application to send IoT data to the cloud in real time.
+
+Real-time data monitoring between Tizen service app and ThingsPark cloud is empowered through ThingsPark plugin.
```

Comment:
Please rephrase to:
Real-time data monitoring between the Tizen service app and the ThingsPark cloud is empowered through the ThingsPark plugin.